test/importers/test_import_swagger.py

```diff
@@ -0,0 +1,89 @@
+import yaml
+
+from sysl.importers.import_swagger import SwaggerTranslator
+from sysl.util import writer
+
+class FakeLogger:
+    def __init__(self):
+        self.warnings = []
+
+    def warn(self, msg):
+        self.warnings.append(msg)
+
+SWAGGER_WITH_ARRAY_TYPE_WITH_EXAMPLE = r"""swagger: "2.0"
+basePath: /fruit-basket
+info:
+    title: Fruit API
+    version: 1.0.0
+definitions:
+    FruitBasket:
+        additionalProperties: false
+        properties:
+            fruit:
+                example: '[{"id":"banana"}, {"id":"mango"}]'
+                items:
+                    type: object
+                type: array
+paths: {}
+"""
+
+SWAGGER_WITH_TYPELESS_ITEMS = r"""swagger: "2.0"
+basePath: /fruit-basket
+info:
+    title: Fruit API
+    version: 1.0.0
+definitions:
+    FruitBasket:
+        additionalProperties: false
+        properties:
+            fruit:
+                items:
+                    type: object
+paths: {}
+"""
+
+SWAGGER_OBJECT_WITH_NO_PROPERTIES = r"""swagger: "2.0"
+basePath: /fruit-basket
+info:
+    title: Fruit API
+    version: 1.0.0
+definitions:
+    MysteriousObject:
+        type: object
+paths: {}
+"""
+
+
+def test_importing_swagger_array_type_with_example_produces_sysl_type():
+    swag = yaml.load(SWAGGER_WITH_ARRAY_TYPE_WITH_EXAMPLE)
+    w = writer.Writer('sysl')
+    t = SwaggerTranslator(logger=FakeLogger())
+    t.translate(swag, appname='', package='', w=w)
+    output = str(w)
+    expected_fragment = '    !type FruitBasket:\n        fruit <: set of {}'
```

Comment:
use `sequence of` instead of `set of`

Reply:
from reading the existing code it is not immediatey obvious how to implement this correctly.

my assumption is that all "set of " string fragmets in this importer are produced by the importer when it encounters a swagger array. hence all "set of " string fragments can be safely replaced with "sequence of " string fragments. proposed change here: https://github.com/anz-bank/sysl/pull/192/commits/ecd04b66a7b994f001d695a4a8ea613107819982